Services/data/devicemodels/scripts/EmptyTank-method.js

```diff
@@ -16,12 +13,19 @@
  * @param previousState  The device state since the last iteration
  * @param previousProperties  The device properties since the last iteration
  */
-
 /*jslint unparam: true*/
 function main(context, previousState, previousProperties) {
 
-    log("Executing JavaScript EmptyTank function.");
+    log("Executing 'EmptyTank' JavaScript method.");
+
+    var newFuelLevel = 0;
+
+    log("Setting fuel level to: " + newFuelLevel);
+
+    var state = {
+        fuellevel: newFuelLevel
+    };
+    updateState(state);
```

Comment:
Can you update the fuel level gradually, like how you are doing in TempIncrease and TempDecrease?